translation/en/RedirectedResources/assets/abdata/adv/scenario/c15/00/0/translation.txt

```diff
@@ -1,10 +1,10 @@
 //
 // Dumped for Koikatsu v5.1 (CharaStudio) by KK_TextDump v1.4.6.0
 //
-「あのー、今いいかな。できれば君と話したいんだけど」=“Umm, is now a good time? I'd like to chat with you, if possible.”
-「……ジー。どうして男がここにいるんだ？」=...*stare*. “Why's there a boy in here?”
-「ああ、最近この学園に転入したんだ。俺は[P名前]って言うんだけど」=“Ah! I just transferred to the school. My name's [P名前].”
-「クンクン、クンクン。怪しいニオイが……」=*sniff, sniff*. “Smells fishy...”
-「え？　俺、何かニオってる？」=“Huh? Do I smell?”
+「あのー、今いいかな。できれば君と話したいんだけど」=“Uhm, is now a good time? I'd like to chat with you, if possible.”
+「……ジー。どうして男がここにいるんだ？」=“...*stare*. Why's there a boy in here?”
+「ああ、最近この学園に転入したんだ。俺は[P名前]って言うんだけど」=“Aah, I transferred in recently. I go by [P名前].”
+「クンクン、クンクン。怪しいニオイが……」=“*sniff sniff*, *sniff sniff*. Smells fishy...”
```

Comment:
This line was translated two ways before, I prefer the `*sniff, sniff*` variant over the `*sniff sniff*, *sniff sniff*` variant.
```suggestion
「クンクン、クンクン。怪しいニオイが……」=“*sniff, sniff* Smells fishy...”
```